docs/cvl/using.md

```diff
@@ -14,6 +14,11 @@ for more information.
 ```{contents}
 ```
 
+Examples
```

Comment:
remove the contents part.
remove the todo

Reply:
done

Reply:
I disagree with removing both of these.
 - The `{contents}` directive is present on every page in this section.  I think we should leave it for consistency, and also because as we finish the documentation it will become more important but we'll forget to add it.
 - The `{todo}` need to indicate that this page is incomplete.  This page is incomplete because it doesn't explain how the `using` statement works or what you can and can't write.